index.html

```diff
@@ -34,31 +34,244 @@
     <!-- ─── Headline Section ────────────────────────────────────────── -->
     <section class="headline">
         <div class="personal-info">
-            <h1>I'm Cosmos Glad to see you!</h1>
+            <h1>I'm Cosmos <br> Glad to see you!</h1>
             <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <div class="socail-media">
+                <h3>Let's connect</h3>
+                <div class="social-icon">
+                    <ul>
+                        <li>
+                            <a href="#"><img src="./images/Vector-3.svg" alt="twitter"></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="./images/Linkedin icon.svg" alt="linked-in"></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="./images/Vector-1.svg" alt="M"></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="./images/Vector-2.svg" alt="github"></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="./images/Vector.svg" alt="sure"></a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+    </section>
+    <!-- ─── End Headline Section ────────────────────────────────────── -->
+
+    <!-- ─── Portfolio Works ─────────────────────────────────────────── -->
+    <section class="works">
+        <div class="grid-container">
+            <!-- ─── Works Project 1 ─────────────────────────── -->
```

Comment:
I must you have done exceptionally well on this project so far. But there are a few things that need to be improved on.
- [ ] Your design is slightly different from the Figma design. See both images below and see the differences. Kindly ensure your project looks almost exactly like the provided template. This skill shows you can follow product requirements with details. 

Your design | Figma Design
---- | ----
![image](https://user-images.githubusercontent.com/84156714/212422818-aa6ca8cc-45a7-4fc2-8ea4-d72cc4e5b16b.png) | ![image](https://user-images.githubusercontent.com/84156714/212423077-62f1b11d-fbfa-4c97-a8f7-2a01fc917ba5.png)

Reply:
Hi @bobb-Rob,
I thank you for making time to review my PR.
Relating to my design please find attached a screenshot of my design for the headline.
I honestly think the design you are seeing might be from your browser because this same design was approved by another reviewer 2days ago, before starting this project. 
 
<img width="388" alt="Screenshot 2023-01-13 at 10 03 10 PM" src="https://user-images.githubusercontent.com/29270468/212427815-433dc5a5-2d4d-42e8-b5d4-40314975e432.png">